src/mono/wasm/runtime/corebindings.c

```diff
@@ -107,25 +103,29 @@ mono_wasm_typed_array_new (char *arr, int length, int size, int type)
 	case MARSHAL_ARRAY_DOUBLE:
 		typeClass = mono_get_double_class();
 		break;
+	default:
+		printf ("Invalid marshal type %d in mono_wasm_typed_array_new", type);
+		abort();
```

Comment:
the interop tests are crashing on this abort that I asked for.  looks like some cases (`UBYTE`, `UBYTE_C`) aren't covered.

Reply:
Sounds reasonable to add them - thanks for suggesting this so we could flush it out.